modules/ROOT/pages/branded-emails.adoc

```diff
@@ -101,11 +101,12 @@ The JSON Web Token (JWT) authentication mechanism creates Anypoint user records
 . Log in to your API community as an administrative user.
 . In *Access Management*, click *Users*.
 . Click the impersonated Anypoint user with the email address of the Salesforce administrator.
+. Add the Organization Administrator role to the user on each organization and suborganizations.
```

Comment:
I'd suggest "... each organization and its suborganizations" or "...each organization and suborganization", depending on your meaning

Reply:
Thanks, I will update this.